common/openapi.yml

```diff
@@ -123,6 +123,14 @@ components:
         workflow_params:
           type: object
           description: Input set
+#        workflow_attachments:
```

Comment:
are these commented lines included here on purpose? (perhaps as a reminder of some kind?)